### PR TITLE
Fix bug and icon for archives (fixes #927)

### LINKED ISSUE #927: JS error when trying to see test details
allure-js-error

stack from console:

```
Uncaught TypeError: Cannot read property 'type' of undefined
    at Object.e.default (app.js:11)
    at Object.main (app.js:11)
    at n (app.js:8)
    at r (app.js:8)
    at Object.l (app.js:8)
    at Object.n [as invokePartial] (app.js:8)
    at 1 (app.js:11)
    at s (app.js:8)
    at n (app.js:8)
    at Object.<anonymous> (app.js:8)
```

We are using Allure 1.4.23.hotfix1 (same error with 1.5.1) , allure-maven-plugin 2.5 and junit adapter.
It's broken suddenly - a lot of old reports are correct.
Unfortunately I cannot attach complete report. 
Can I somehow disable minification of the app.js file?

### PR DESCRIPTION
**Pull request by sseliverstov**

### CODE REVIEW

File: allure-report-face/src/util/attachmentType.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import d3 from 'd3';
+import d3 from 'd3';
```
Comment (just-boris) [inline allure-report-face/src/util/attachmentType.js]: What is there? I see unpritable symbol.

**Comment by just-boris** [comment]
If you are using IDEA, I'd recommend you to enable this option, to see errors directly in your IDE.
image

For some other editors, you may also look for ESlint plugin, to see errors instanly

**Comment by sseliverstov** [comment]
Tanks, Boris!

**Comment by just-boris** [comment]
When you are opening ZIP attachement, the save dialog immediately pops up
http://ci.qatools.ru/job/allure1_pull-request/45/Allure_report/index.html#xUnit/63b4b77b0e86782b/a25c24fc62520ca6/b55afac94f791610

I don't think that it is a good for user experience. I'd suggest to create a special view for this kind of attachement, like this page https://github.com/allure-framework/allure1/blob/927-unknown-attaches-brocken/allure-model/src/test/resources/allure-results/00a0bc5a-6372-4e66-8088-1571966a156b-attachment.tar.gz

WDYT?

**Comment by sseliverstov** [comment]
Good idea, but may be in separated issue?

**Comment by baev** [comment]
> I don't think that it is a good for user experience. I'd suggest to create a special view for this kind of attachement

@sseliverstov @just-boris some of you guys please clean an issue for that

BTW panzeren!

**Pull request merged**